app/commands/submission/create.rb

```diff
@@ -102,11 +101,6 @@ def create_files!
       end
     end
 
-    def update_solution!
-      solution.status = :submitted if solution.pending?
-      solution.save!
-    end
```

Comment:
This never actually got used, and I don't have plans for it now, so I'm removing the field altogether.